routes/web_legacy.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+// -- Legacy links
+Route::redirect('/sk/uvod/kontakt.html', '/oddelenie-architektury');
```

Comment:
I would consider permanent redirect (to tell google to index new url) for all legacy routes.

```suggestion
Route::redirect('/sk/uvod/kontakt.html', '/oddelenie-architektury', 301);
```

But we can do it in the next step - after we check it covers urls properly.

Reply:
@igor-kamil 
> where do you handle stripping index.php/ from URL?

As far as I understand, this stripping happens automatically in `php artisan serve` (dev server) and also in Nginx with the 'canonical' Nginx configuration: https://laravel.com/docs/7.x/deployment#nginx

I've tested this in dev and on regarch.webumenia.sk and it worked for me. But feel free to try it for yourself as I was testing it quite frantically 😅

Reply:
I use [valet](https://laravel.com/docs/8.x/valet) for local development (nginx instead of `php artisan serve`) and it doesn't work like that for me. that `try_files $uri $uri/ /index.php?$query_string;` is just telling to pass whole uri as parameter for `index.php` - not about removing it from url. but it's possible to cover removing `index.php` in nginx virtual host config - so if you wanted to do it this way, I am ok with that. 
Once I already had to deal with the similar scenario and I ended up adding `removeIndexPhpFromUrl()` to `RouteServiceProvider` as covered in https://ma.ttias.be/remove-index-php-from-the-url-in-laravel/

Reply:
sorry. only now I read properly - if it works on regarch.webumenia.sk - I have no further objections ;)

Reply:
so I tested it on regarch.webumenia.sk and you are right -> it works. however it produces "ugly" url with remaining `index.php` inside - e.g. http://www.register-architektury.sk/index.php/sk/objekty/566-administratívna-budova-fondu-národnej-obnovy.html -> https://regarch.webumenia.sk/index.php/sk/objekt/116-administrativna-budova-fondu-narodnej-obnovy
so I propose to remove the `index.php` from url in `RouteServiceProvider` anyway.